spec/unit/puppet/x/jenkins/provider/cli_spec.rb

```diff
@@ -478,15 +467,15 @@
 
     context 'network failure' do
       context 'without ssh_private_key' do
-        CLI_NET_ERRORS.each do |error|
+        CLI_AUTH_ERRORS.each do |error|
           it 'does not retry cli on AuthError exception' do
```

Comment:
In this test I went with this description of what it was supposed to do rather than the implementation.